src/index.js

```diff
@@ -41,6 +41,15 @@ async function isWithinRange(forwardedFor, fastlyPublicIps, whitelistedIps = '')
     .some((myranges) => (range.isRange ? range.inRange(ip, myranges) : range === ip)));
 }
 
+async function getEmbedKind(domainList) {
+  const cpy = domainList;
+  // remove first level domain
+  cpy.pop();
+  return cpy
+    .filter((domain) => domain !== 'www')
+    .join('-');
```

Comment:
I have 2 change proposals:
- turn the order around, so `spark.adobe.com` results in `embed-adobe-spark`
- add the 3rd level class as an additonal class: `spark.adobe.com` generated `embed-adobe embed-adobe-spark`. Web developers would be able to cascade common styles for all `embed-adobe` elements, but only override some styles for `embed-adobe-spark` elements.